packages/smooth_app/lib/pages/hunger_games/congrats.dart

```diff
@@ -28,20 +28,25 @@ class CongratsWidget extends StatelessWidget {
     final UserManagementProvider userManagementProvider =
         context.watch<UserManagementProvider>();
 
+    final Brightness brightness = Theme.of(context).brightness;
+    final bool isDarkMode = brightness == Brightness.dark;
+
     return Center(
       child: Column(
         mainAxisAlignment: MainAxisAlignment.center,
         children: <Widget>[
           const Icon(
             Icons.grade,
             color: Colors.amber,
-            size: 100,
+            size: 70,
           ),
           Padding(
             padding: const EdgeInsets.symmetric(vertical: MEDIUM_SPACE),
             child: Text(
               appLocalizations.thanks_for_contributing,
-              style: Theme.of(context).textTheme.bodyText1,
+              style: Theme.of(context).textTheme.bodyText1!.apply(
+                    color: isDarkMode ? Colors.white : Colors.black,
+                  ),
```

Comment:
Are those changes necessary, this will likely look out of place in the coming AMOLED mode